docs/build/autogenerate.rst

```diff
@@ -778,7 +787,7 @@
         autopep8.entrypoint = autopep8
         autopep8.options = --in-place REVISION_SCRIPT_FILENAME
 
-* ``cwd`` - optional working directory from which the console script is run.
+* ``cwd`` - optional working directory from which the formatting tool is run.
```

Comment:
**mike bayer** (zzzeek) wrote:

"code processing tool"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4788